Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -217,4 +217,14 @@ - (void)configureWithData:(id)data {
     NSAssert(false, @"BPKCalendarCell configureWithData: should be overridden. The base class method should never be called.");
 }
 
+- (NSString *)formattedDate:(NSDate *)date locale:(NSLocale *)locale formatter:(NSDateFormatter *)formatter {
+    formatter.locale = locale;
+    formatter.dateStyle = NSDateFormatterLongStyle;
```

Comment:
Shouldn't this also be the responsibility of the calendar not the cell?

Reply:
![of course](https://media.giphy.com/media/J1M6o5U4M7LmoSYEYl/giphy.gif)